### PR TITLE
Enabling TSO and GSO for network tests.

### PR DESCRIPTION
**Pull request by manvanthar**
Enabling TSO and GSO for network tests.

Signed-off-by: Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
Fixed all the review comment of PR #1451

**Comment by manvanthar** [comment]
avocado list network_test.py
Failed to load plugin from module "avocado_runner_vm": DistributionNotFound(Requirement.parse('libvirt-python'), None) :
  File "/usr/local/lib/python3.6/site-packages/avocado_framework-72.0-py3.6.egg/avocado/core/extension_manager.py", line 62, in __init__
    plugin = ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2317, in load
    self.require(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2340, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 774, in resolve
    raise DistributionNotFound(req, requirers)

INSTRUMENTED network_test.py:NetworkTest.test_gro
INSTRUMENTED network_test.py:NetworkTest.test_gso
INSTRUMENTED network_test.py:NetworkTest.test_lro
INSTRUMENTED network_test.py:NetworkTest.test_tso
INSTRUMENTED network_test.py:NetworkTest.test_ping
INSTRUMENTED network_test.py:NetworkTest.test_floodping
INSTRUMENTED network_test.py:NetworkTest.test_ssh
INSTRUMENTED network_test.py:NetworkTest.test_scp
INSTRUMENTED network_test.py:NetworkTest.test_jumbo_frame
INSTRUMENTED network_test.py:NetworkTest.test_statistics
INSTRUMENTED network_test.py:NetworkTest.test_promisc


avocado run network_test.py -m network_test.py.data/network_test.yaml 
Failed to load plugin from module "avocado_runner_vm": DistributionNotFound(Requirement.parse('libvirt-python'), None) :
  File "/usr/local/lib/python3.6/site-packages/avocado_framework-72.0-py3.6.egg/avocado/core/extension_manager.py", line 62, in __init__
    plugin = ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2317, in load
    self.require(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2340, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 774, in resolve
    raise DistributionNotFound(req, requirers)

JOB ID     : 7e8ac7e4ce189cb71d345e65dddcbfeb1a8e88b1
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-29T17.54-7e8ac7e/job.log
 (01/99) network_test.py:NetworkTest.test_gro;1500-d20e: PASS (11.77 s)
 (02/99) network_test.py:NetworkTest.test_gro;2000-3295: PASS (12.54 s)
 (03/99) network_test.py:NetworkTest.test_gro;3000-8d15: PASS (12.47 s)
 (04/99) network_test.py:NetworkTest.test_gro;4000-a8bf: PASS (12.45 s)
 (05/99) network_test.py:NetworkTest.test_gro;5000-fe4f: PASS (12.56 s)
 (06/99) network_test.py:NetworkTest.test_gro;6000-da64: PASS (12.55 s)
 (07/99) network_test.py:NetworkTest.test_gro;7000-354d: PASS (12.49 s)
 (08/99) network_test.py:NetworkTest.test_gro;8000-257a: PASS (12.46 s)
 (09/99) network_test.py:NetworkTest.test_gro;9000-74de: PASS (12.43 s)
 (10/99) network_test.py:NetworkTest.test_gso;1500-d20e: PASS (11.88 s)
 (11/99) network_test.py:NetworkTest.test_gso;2000-3295: PASS (12.47 s)
 (12/99) network_test.py:NetworkTest.test_gso;3000-8d15: PASS (12.53 s)
 (13/99) network_test.py:NetworkTest.test_gso;4000-a8bf: PASS (12.56 s)
 (14/99) network_test.py:NetworkTest.test_gso;5000-fe4f: PASS (12.49 s)
 (15/99) network_test.py:NetworkTest.test_gso;6000-da64: PASS (12.45 s)
 (16/99) network_test.py:NetworkTest.test_gso;7000-354d: PASS (12.45 s)
 (17/99) network_test.py:NetworkTest.test_gso;8000-257a: PASS (12.45 s)
 (18/99) network_test.py:NetworkTest.test_gso;9000-74de: PASS (12.49 s)
 (19/99) network_test.py:NetworkTest.test_lro;1500-d20e: FAIL: lro on failed (7.74 s)
 (20/99) network_test.py:NetworkTest.test_lro;2000-3295: FAIL: lro on failed (8.47 s)
 (21/99) network_test.py:NetworkTest.test_lro;3000-8d15: FAIL: lro on failed (8.39 s)
 (22/99) network_test.py:NetworkTest.test_lro;4000-a8bf: FAIL: lro on failed (8.52 s)
 (23/99) network_test.py:NetworkTest.test_lro;5000-fe4f: FAIL: lro on failed (8.46 s)
 (24/99) network_test.py:NetworkTest.test_lro;6000-da64: FAIL: lro on failed (8.48 s)
 (25/99) network_test.py:NetworkTest.test_lro;7000-354d: FAIL: lro on failed (8.46 s)
 (26/99) network_test.py:NetworkTest.test_lro;8000-257a: FAIL: lro on failed (8.41 s)
 (27/99) network_test.py:NetworkTest.test_lro;9000-74de: FAIL: lro on failed (8.39 s)
 (28/99) network_test.py:NetworkTest.test_tso;1500-d20e: PASS (11.74 s)
 (29/99) network_test.py:NetworkTest.test_tso;2000-3295: PASS (12.59 s)
 (30/99) network_test.py:NetworkTest.test_tso;3000-8d15: PASS (12.44 s)
 (31/99) network_test.py:NetworkTest.test_tso;4000-a8bf: PASS (12.54 s)
 (32/99) network_test.py:NetworkTest.test_tso;5000-fe4f: PASS (12.55 s)
 (33/99) network_test.py:NetworkTest.test_tso;6000-da64: PASS (12.53 s)
 (34/99) network_test.py:NetworkTest.test_tso;7000-354d: PASS (12.57 s)
 (35/99) network_test.py:NetworkTest.test_tso;8000-257a: PASS (12.48 s)
 (36/99) network_test.py:NetworkTest.test_tso;9000-74de: PASS (12.48 s)
 (37/99) network_test.py:NetworkTest.test_ping;1500-d20e: PASS (7.50 s)
 (38/99) network_test.py:NetworkTest.test_ping;2000-3295: PASS (8.26 s)
 (39/99) network_test.py:NetworkTest.test_ping;3000-8d15: PASS (8.26 s)
 (40/99) network_test.py:NetworkTest.test_ping;4000-a8bf: PASS (8.24 s)
 (41/99) network_test.py:NetworkTest.test_ping;5000-fe4f: PASS (8.24 s)
 (42/99) network_test.py:NetworkTest.test_ping;6000-da64: PASS (8.27 s)
 (43/99) network_test.py:NetworkTest.test_ping;7000-354d: PASS (8.16 s)
 (44/99) network_test.py:NetworkTest.test_ping;8000-257a: PASS (8.29 s)
 (45/99) network_test.py:NetworkTest.test_ping;9000-74de: PASS (8.27 s)
 (46/99) network_test.py:NetworkTest.test_floodping;1500-d20e: PASS (3.46 s)
 (47/99) network_test.py:NetworkTest.test_floodping;2000-3295: PASS (4.19 s)
 (48/99) network_test.py:NetworkTest.test_floodping;3000-8d15: PASS (4.21 s)
 (49/99) network_test.py:NetworkTest.test_floodping;4000-a8bf: PASS (4.12 s)
 (50/99) network_test.py:NetworkTest.test_floodping;5000-fe4f: PASS (4.12 s)
 (51/99) network_test.py:NetworkTest.test_floodping;6000-da64: PASS (4.06 s)
 (52/99) network_test.py:NetworkTest.test_floodping;7000-354d: PASS (4.14 s)
 (53/99) network_test.py:NetworkTest.test_floodping;8000-257a: PASS (4.16 s)
 (54/99) network_test.py:NetworkTest.test_floodping;9000-74de: PASS (4.13 s)
 (55/99) network_test.py:NetworkTest.test_ssh;1500-d20e: PASS (3.59 s)
 (56/99) network_test.py:NetworkTest.test_ssh;2000-3295: PASS (4.30 s)
 (57/99) network_test.py:NetworkTest.test_ssh;3000-8d15: PASS (4.32 s)
 (58/99) network_test.py:NetworkTest.test_ssh;4000-a8bf: PASS (4.26 s)
 (59/99) network_test.py:NetworkTest.test_ssh;5000-fe4f: PASS (4.35 s)
 (60/99) network_test.py:NetworkTest.test_ssh;6000-da64: PASS (4.35 s)
 (61/99) network_test.py:NetworkTest.test_ssh;7000-354d: PASS (4.37 s)
 (62/99) network_test.py:NetworkTest.test_ssh;8000-257a: PASS (4.29 s)
 (63/99) network_test.py:NetworkTest.test_ssh;9000-74de: PASS (4.42 s)
 (64/99) network_test.py:NetworkTest.test_scp;1500-d20e: PASS (18.37 s)
 (65/99) network_test.py:NetworkTest.test_scp;2000-3295: PASS (19.95 s)
 (66/99) network_test.py:NetworkTest.test_scp;3000-8d15: PASS (19.56 s)
 (67/99) network_test.py:NetworkTest.test_scp;4000-a8bf: PASS (19.45 s)
 (68/99) network_test.py:NetworkTest.test_scp;5000-fe4f: PASS (19.52 s)
 (69/99) network_test.py:NetworkTest.test_scp;6000-da64: PASS (19.29 s)
 (70/99) network_test.py:NetworkTest.test_scp;7000-354d: PASS (19.67 s)
 (71/99) network_test.py:NetworkTest.test_scp;8000-257a: PASS (19.49 s)
 (72/99) network_test.py:NetworkTest.test_scp;9000-74de: PASS (19.54 s)
 (73/99) network_test.py:NetworkTest.test_jumbo_frame;1500-d20e: PASS (6.54 s)
 (74/99) network_test.py:NetworkTest.test_jumbo_frame;2000-3295: PASS (7.35 s)
 (75/99) network_test.py:NetworkTest.test_jumbo_frame;3000-8d15: PASS (7.29 s)
 (76/99) network_test.py:NetworkTest.test_jumbo_frame;4000-a8bf: PASS (7.35 s)
 (77/99) network_test.py:NetworkTest.test_jumbo_frame;5000-fe4f: PASS (7.17 s)
 (78/99) network_test.py:NetworkTest.test_jumbo_frame;6000-da64: PASS (7.35 s)
 (79/99) network_test.py:NetworkTest.test_jumbo_frame;7000-354d: PASS (7.25 s)
 (80/99) network_test.py:NetworkTest.test_jumbo_frame;8000-257a: PASS (7.26 s)
 (81/99) network_test.py:NetworkTest.test_jumbo_frame;9000-74de: PASS (7.34 s)
 (82/99) network_test.py:NetworkTest.test_statistics;1500-d20e: PASS (7.53 s)
 (83/99) network_test.py:NetworkTest.test_statistics;2000-3295: PASS (8.15 s)
 (84/99) network_test.py:NetworkTest.test_statistics;3000-8d15: PASS (8.18 s)
 (85/99) network_test.py:NetworkTest.test_statistics;4000-a8bf: PASS (8.15 s)
 (86/99) network_test.py:NetworkTest.test_statistics;5000-fe4f: PASS (8.20 s)
 (87/99) network_test.py:NetworkTest.test_statistics;6000-da64: PASS (8.20 s)
 (88/99) network_test.py:NetworkTest.test_statistics;7000-354d: PASS (8.13 s)
 (89/99) network_test.py:NetworkTest.test_statistics;8000-257a: PASS (8.30 s)
 (90/99) network_test.py:NetworkTest.test_statistics;9000-74de: PASS (8.21 s)
 (91/99) network_test.py:NetworkTest.test_promisc;1500-d20e: PASS (11.72 s)
 (92/99) network_test.py:NetworkTest.test_promisc;2000-3295: PASS (12.42 s)
 (93/99) network_test.py:NetworkTest.test_promisc;3000-8d15: PASS (12.39 s)
 (94/99) network_test.py:NetworkTest.test_promisc;4000-a8bf: PASS (12.41 s)
 (95/99) network_test.py:NetworkTest.test_promisc;5000-fe4f: PASS (12.41 s)
 (96/99) network_test.py:NetworkTest.test_promisc;6000-da64: PASS (12.48 s)
 (97/99) network_test.py:NetworkTest.test_promisc;7000-354d: PASS (12.48 s)
 (98/99) network_test.py:NetworkTest.test_promisc;8000-257a: PASS (12.38 s)
 (99/99) network_test.py:NetworkTest.test_promisc;9000-74de: PASS (12.42 s)
RESULTS    : PASS 90 | ERROR 0 | FAIL 9 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 986.04 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-29T17.54-7e8ac7e/results.html

**Comment by manvanthar** [comment]
[job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3905266/job.log)

**Comment by manvanthar** [comment]
Run log for the backport with higher Network packets.

avocado run network_test.py -m network_test.py.data/network_test.yaml 
Failed to load plugin from module "avocado_runner_vm": DistributionNotFound(Requirement.parse('libvirt-python'), None) :
  File "/usr/local/lib/python3.6/site-packages/avocado_framework-72.0-py3.6.egg/avocado/core/extension_manager.py", line 62, in __init__
    plugin = ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2317, in load
    self.require(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2340, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 774, in resolve
    raise DistributionNotFound(req, requirers)

JOB ID     : 288da7de2877828c80864844bf3199ecf8573f0a
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-29T18.35-288da7d/job.log
 (01/99) network_test.py:NetworkTest.test_gro;1500-d20e: PASS (48.90 s)
 (02/99) network_test.py:NetworkTest.test_gro;2000-3295: PASS (57.05 s)
 (03/99) network_test.py:NetworkTest.test_gro;3000-8d15: PASS (48.36 s)
 (04/99) network_test.py:NetworkTest.test_gro;4000-a8bf: PASS (50.61 s)
 (05/99) network_test.py:NetworkTest.test_gro;5000-fe4f: PASS (66.73 s)
 (06/99) network_test.py:NetworkTest.test_gro;6000-da64: PASS (47.03 s)
 (07/99) network_test.py:NetworkTest.test_gro;7000-354d: PASS (57.11 s)
 (08/99) network_test.py:NetworkTest.test_gro;8000-257a: PASS (58.89 s)
 (09/99) network_test.py:NetworkTest.test_gro;9000-74de: PASS (48.49 s)
 (10/99) network_test.py:NetworkTest.test_gso;1500-d20e: PASS (52.50 s)
 (11/99) network_test.py:NetworkTest.test_gso;2000-3295: PASS (58.58 s)
 (12/99) network_test.py:NetworkTest.test_gso;3000-8d15: PASS (47.69 s)
 (13/99) network_test.py:NetworkTest.test_gso;4000-a8bf: PASS (48.40 s)
 (14/99) network_test.py:NetworkTest.test_gso;5000-fe4f: PASS (55.23 s)
 (15/99) network_test.py:NetworkTest.test_gso;6000-da64: PASS (52.41 s)
 (16/99) network_test.py:NetworkTest.test_gso;7000-354d: PASS (53.81 s)
 (17/99) network_test.py:NetworkTest.test_gso;8000-257a: PASS (52.32 s)
 (18/99) network_test.py:NetworkTest.test_gso;9000-74de: PASS (60.18 s)
 (19/99) network_test.py:NetworkTest.test_lro;1500-d20e: FAIL: lro on failed (32.42 s)
 (20/99) network_test.py:NetworkTest.test_lro;2000-3295: FAIL: lro on failed (27.82 s)
 (21/99) network_test.py:NetworkTest.test_lro;3000-8d15: FAIL: lro on failed (28.30 s)
 (22/99) network_test.py:NetworkTest.test_lro;4000-a8bf: FAIL: lro on failed (27.53 s)
 (23/99) network_test.py:NetworkTest.test_lro;5000-fe4f: FAIL: lro on failed (26.85 s)
 (24/99) network_test.py:NetworkTest.test_lro;6000-da64: FAIL: lro on failed (27.20 s)
 (25/99) network_test.py:NetworkTest.test_lro;7000-354d: FAIL: lro on failed (27.56 s)
 (26/99) network_test.py:NetworkTest.test_lro;8000-257a: FAIL: lro on failed (29.03 s)
 (27/99) network_test.py:NetworkTest.test_lro;9000-74de: FAIL: lro on failed (26.55 s)
 (28/99) network_test.py:NetworkTest.test_tso;1500-d20e: PASS (49.30 s)
 (29/99) network_test.py:NetworkTest.test_tso;2000-3295: PASS (48.76 s)
 (30/99) network_test.py:NetworkTest.test_tso;3000-8d15: PASS (48.79 s)
 (31/99) network_test.py:NetworkTest.test_tso;4000-a8bf: PASS (53.15 s)
 (32/99) network_test.py:NetworkTest.test_tso;5000-fe4f: PASS (48.82 s)
 (33/99) network_test.py:NetworkTest.test_tso;6000-da64: PASS (50.58 s)
 (34/99) network_test.py:NetworkTest.test_tso;7000-354d: PASS (48.53 s)
 (35/99) network_test.py:NetworkTest.test_tso;8000-257a: PASS (51.34 s)
 (36/99) network_test.py:NetworkTest.test_tso;9000-74de: PASS (53.06 s)
 (37/99) network_test.py:NetworkTest.test_ping;1500-d20e: PASS (15.86 s)
 (38/99) network_test.py:NetworkTest.test_ping;2000-3295: PASS (16.61 s)
 (39/99) network_test.py:NetworkTest.test_ping;3000-8d15: PASS (16.63 s)
 (40/99) network_test.py:NetworkTest.test_ping;4000-a8bf: PASS (16.53 s)
 (41/99) network_test.py:NetworkTest.test_ping;5000-fe4f: PASS (16.63 s)
 (42/99) network_test.py:NetworkTest.test_ping;6000-da64: PASS (16.63 s)
 (43/99) network_test.py:NetworkTest.test_ping;7000-354d: PASS (16.52 s)
 (44/99) network_test.py:NetworkTest.test_ping;8000-257a: PASS (16.52 s)
 (45/99) network_test.py:NetworkTest.test_ping;9000-74de: PASS (16.54 s)
 (46/99) network_test.py:NetworkTest.test_floodping;1500-d20e: PASS (28.18 s)
 (47/99) network_test.py:NetworkTest.test_floodping;2000-3295: PASS (30.60 s)
 (48/99) network_test.py:NetworkTest.test_floodping;3000-8d15: PASS (28.10 s)
 (49/99) network_test.py:NetworkTest.test_floodping;4000-a8bf: PASS (27.21 s)
 (50/99) network_test.py:NetworkTest.test_floodping;5000-fe4f: PASS (28.02 s)
 (51/99) network_test.py:NetworkTest.test_floodping;6000-da64: PASS (27.64 s)
 (52/99) network_test.py:NetworkTest.test_floodping;7000-354d: PASS (28.73 s)
 (53/99) network_test.py:NetworkTest.test_floodping;8000-257a: PASS (27.15 s)
 (54/99) network_test.py:NetworkTest.test_floodping;9000-74de: PASS (28.36 s)
 (55/99) network_test.py:NetworkTest.test_ssh;1500-d20e: PASS (6.73 s)
 (56/99) network_test.py:NetworkTest.test_ssh;2000-3295: PASS (7.46 s)
 (57/99) network_test.py:NetworkTest.test_ssh;3000-8d15: PASS (7.51 s)
 (58/99) network_test.py:NetworkTest.test_ssh;4000-a8bf: PASS (7.51 s)
 (59/99) network_test.py:NetworkTest.test_ssh;5000-fe4f: PASS (7.40 s)
 (60/99) network_test.py:NetworkTest.test_ssh;6000-da64: PASS (7.40 s)
 (61/99) network_test.py:NetworkTest.test_ssh;7000-354d: PASS (7.47 s)
 (62/99) network_test.py:NetworkTest.test_ssh;8000-257a: PASS (7.47 s)
 (63/99) network_test.py:NetworkTest.test_ssh;9000-74de: PASS (7.51 s)
 (64/99) network_test.py:NetworkTest.test_scp;1500-d20e: PASS (22.31 s)
 (65/99) network_test.py:NetworkTest.test_scp;2000-3295: PASS (23.19 s)
 (66/99) network_test.py:NetworkTest.test_scp;3000-8d15: PASS (22.52 s)
 (67/99) network_test.py:NetworkTest.test_scp;4000-a8bf: PASS (22.55 s)
 (68/99) network_test.py:NetworkTest.test_scp;5000-fe4f: PASS (22.41 s)
 (69/99) network_test.py:NetworkTest.test_scp;6000-da64: PASS (22.61 s)
 (70/99) network_test.py:NetworkTest.test_scp;7000-354d: PASS (22.62 s)
 (71/99) network_test.py:NetworkTest.test_scp;8000-257a: PASS (22.99 s)
 (72/99) network_test.py:NetworkTest.test_scp;9000-74de: PASS (22.41 s)
 (73/99) network_test.py:NetworkTest.test_jumbo_frame;1500-d20e: PASS (9.70 s)
 (74/99) network_test.py:NetworkTest.test_jumbo_frame;2000-3295: PASS (10.41 s)
 (75/99) network_test.py:NetworkTest.test_jumbo_frame;3000-8d15: PASS (10.33 s)
 (76/99) network_test.py:NetworkTest.test_jumbo_frame;4000-a8bf: PASS (10.40 s)
 (77/99) network_test.py:NetworkTest.test_jumbo_frame;5000-fe4f: PASS (10.48 s)
 (78/99) network_test.py:NetworkTest.test_jumbo_frame;6000-da64: PASS (10.33 s)
 (79/99) network_test.py:NetworkTest.test_jumbo_frame;7000-354d: PASS (10.29 s)
 (80/99) network_test.py:NetworkTest.test_jumbo_frame;8000-257a: PASS (10.37 s)
 (81/99) network_test.py:NetworkTest.test_jumbo_frame;9000-74de: PASS (10.40 s)
 (82/99) network_test.py:NetworkTest.test_statistics;1500-d20e: PASS (28.43 s)
 (83/99) network_test.py:NetworkTest.test_statistics;2000-3295: PASS (27.39 s)
 (84/99) network_test.py:NetworkTest.test_statistics;3000-8d15: PASS (31.95 s)
 (85/99) network_test.py:NetworkTest.test_statistics;4000-a8bf: PASS (27.29 s)
 (86/99) network_test.py:NetworkTest.test_statistics;5000-fe4f: PASS (46.26 s)
 (87/99) network_test.py:NetworkTest.test_statistics;6000-da64: PASS (26.82 s)
 (88/99) network_test.py:NetworkTest.test_statistics;7000-354d: PASS (35.47 s)
 (89/99) network_test.py:NetworkTest.test_statistics;8000-257a: PASS (27.27 s)
 (90/99) network_test.py:NetworkTest.test_statistics;9000-74de: PASS (25.77 s)
 (91/99) network_test.py:NetworkTest.test_promisc;1500-d20e: PASS (14.25 s)
 (92/99) network_test.py:NetworkTest.test_promisc;2000-3295: PASS (15.41 s)
 (93/99) network_test.py:NetworkTest.test_promisc;3000-8d15: PASS (15.60 s)
 (94/99) network_test.py:NetworkTest.test_promisc;4000-a8bf: PASS (15.50 s)
 (95/99) network_test.py:NetworkTest.test_promisc;5000-fe4f: PASS (15.58 s)
 (96/99) network_test.py:NetworkTest.test_promisc;6000-da64: PASS (15.14 s)
 (97/99) network_test.py:NetworkTest.test_promisc;7000-354d: PASS (14.68 s)
 (98/99) network_test.py:NetworkTest.test_promisc;8000-257a: PASS (15.38 s)
 (99/99) network_test.py:NetworkTest.test_promisc;9000-74de: PASS (15.43 s)
RESULTS    : PASS 90 | ERROR 0 | FAIL 9 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 2851.86 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-29T18.35-288da7d/results.html

**Comment by manvanthar** [comment]
@narasimhan-v the job.log file is >10MB for second commit and i am not able to attach it here.

File: io/net/network_test.py [hunk 1]
```diff
@@ -72,57 +72,81 @@ def setUp(self):
...
+        if not HostInfo.ping_check(self, self.iface, self.peer, "5"):
             self.cancel("No connection to peer")
 
     def mtu_set(self):
         '''
         set mtu size
         '''
         if not self.peerinfo.set_mtu_peer(self.peer_interface, self.mtu):
-            self.cancel("Failed to set mtu in peer")
+            self.fail("Failed to set mtu in peer")
```
[the remark this answers is not in the document]
Comment (manvanthar) [reply io/net/network_test.py]: @narasimhan-v ,  Failing to set an MTU is an failure as against to skipping the test.

[the remark this answers is not in the document]
Comment (narasimhan-v) [reply io/net/network_test.py]: @manvanthar This is called from setup, so it was `cancel` earlier, and it still can be cancelled. MTU set is a prerequisite for the actual tests in this test script, So cancel makes sense too.

[the remark this answers is not in the document]
Comment (manvanthar) [reply io/net/network_test.py]: @narasimhan-v ok the changes related to self.fail has been reverted back to self.cancel.

**Pull request merged**